composer.json

```diff
@@ -16,49 +16,50 @@
-        "setono/doctrine-orm-batcher-bundle": "^0.3.1",
+        "setono/doctrine-orm-batcher-bundle": "^0.3.4",
         "setono/symfony-main-request-trait": "^1.0",
         "spatie/enum": "^3.7",
         "sylius/resource-bundle": "^1.6",
-        "symfony/config": "^4.4 || ^5.4",
-        "symfony/console": "^4.4 || ^5.4",
-        "symfony/dependency-injection": "^4.4 || ^5.4",
-        "symfony/event-dispatcher": "^4.4 || ^5.4",
+        "symfony/config": "^5.4 || ^6.0",
+        "symfony/console": "^5.4 || ^6.0",
+        "symfony/dependency-injection": "^5.4 || ^6.0",
+        "symfony/event-dispatcher": "^5.4 || ^6.0",
         "symfony/event-dispatcher-contracts": "^1.1 || ^2.2",
-        "symfony/form": "^4.4 || ^5.4",
-        "symfony/http-foundation": "^4.4 || ^5.4.7",
-        "symfony/http-kernel": "^4.4 || ^5.4",
-        "symfony/messenger": "^4.4 || ^5.4",
-        "symfony/mime": "^4.4 || ^5.4",
-        "symfony/options-resolver": "^4.4 || ^5.4",
-        "symfony/routing": "^4.4 || ^5.4",
-        "symfony/serializer": "^4.4 || ^5.4",
+        "symfony/form": "^5.4 || ^6.0",
+        "symfony/http-foundation": "^5.4.7 || ^6.0",
+        "symfony/http-kernel": "^5.4 || ^6.0",
+        "symfony/mailer": "^5.4 || ^6.0",
+        "symfony/messenger": "^5.4 || ^6.0",
+        "symfony/mime": "^5.4 || ^6.0",
+        "symfony/options-resolver": "^5.4 || ^6.0",
+        "symfony/routing": "^5.4 || ^6.0",
+        "symfony/serializer": "^5.4 || ^6.0",
         "symfony/translation-contracts": "^1.1 || ^2.3",
-        "symfony/uid": "^5.4",
-        "symfony/validator": "^4.4 || ^5.4",
-        "symfony/workflow": "^4.4 || ^5.4",
+        "symfony/uid": "^5.4 || ^6.0",
+        "symfony/validator": "^5.4 || ^6.0",
+        "symfony/workflow": "^5.4 || ^6.0",
         "twig/twig": "^2.14",
         "webmozart/assert": "^1.1"
     },
     "require-dev": {
         "api-platform/core": "^2.6",
-        "friendsofsymfony/oauth-server-bundle": ">2.0.0-alpha.0 ^2.0@dev",
         "lexik/jwt-authentication-bundle": "^2.15",
         "php-http/message-factory": "^1.1",
         "phpspec/phpspec": "^7.2",
         "phpunit/phpunit": "^9.5",
         "polishsymfonycommunity/symfony-mocker-container": "^1.0",
-        "psalm/plugin-phpunit": "^0.16",
-        "psalm/plugin-symfony": "^3.0",
+        "psalm/plugin-phpunit": "^0.18",
+        "psalm/plugin-symfony": "^5.0",
         "roave/security-advisories": "dev-latest",
-        "setono/code-quality-pack": "^2.2",
+        "setono/code-quality-pack": "^2.4",
         "setono/sylius-behat-pack": "^0.2",
-        "sylius/admin-api-bundle": "^1.11",
-        "sylius/sylius": "~1.10.14",
-        "symfony/debug-bundle": "^4.4 || ^5.4",
-        "symfony/dotenv": "^4.4 || ^5.4",
-        "symfony/intl": "^4.4 || ^5.4",
-        "symfony/web-profiler-bundle": "^4.4 || ^5.4",
+        "sylius/mailer-bundle": "^1.8 || ^2.0",
+        "sylius/sylius": "^1.11",
```

Comment:
```suggestion
        "sylius/sylius": "~1.11.15",
```

Reply:
Why remove support for Sylius 1.12?

Reply:
Okay, let's keep it here for now. I will change it later to `~1.12.x`